testProjects/android-test/app/src/main/res/mipmap/ic_launcher_round.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@drawable/ic_launcher_background" />
```

Comment:
If we do not reference `@drawable/ic_launcher_xxx`, can those be removed as well?

Reply:
I guess we can remove all the resource files as we are not querying those in any unit tests.

Reply:
Removed in https://github.com/microsoft/build-server-for-gradle/pull/173/commits/04ee668f23c8e64397b647fe3d63a92636eba3ef

Reply:
Ok, cool!